test-spicy-features

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env bash
+
+BASE=$(cd $(dirname $0) && pwd)
```

Comment:
Let's make sure to quote all user-provided variables, e.g., here
```suggestion
BASE=$(cd "$(dirname "$0")" && pwd)
```

`shellcheck` should point out the instances where this is needed.